dist/tools/ci/print_environment.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+SCRIPTPATH="$( cd "$(dirname "$0")" ; pwd -P )"
+python3 $SCRIPTPATH/print_python_versions.py
```

Comment:
It seems that this file doesn't exist, it should be `print_environment.py`. So this script is broken in the current state.

Reply:
ah damn, this was correct before squash 😢